prometheus/rootfs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM prom/prometheus:0.16.1
+FROM prom/prometheus:master
```

Comment:
Unfortunately only master has the prometheus version we need - they made big changes to the k8s service discovery and it breaks on anything else. Ill make an issue to overhaul the dockerfiles.